src/site/topic8.rst

```diff
@@ -302,16 +302,6 @@ Playing
     System.out.println(myStack.isEmpty());
     System.out.println(myStack);
 
-    // Test expand capacity
-    myStack.push(10);
```

Comment:
imo, this is reasonable to keep, though dumping the comment definitely makes sense